warehouse/models/mart/transit_database/_mart_transit_database.yml

```diff
@@ -487,16 +468,14 @@ models:
           * "Regional Precursor Feed" - This category applies to agencies whose data appears in a
             combined regional feed but who produce some other independent dataset.
             The independent dataset may be an input into the combined regional feed.
-      - name: agency_id
```

Comment:
These columns should be here 👀 If they got removed I think we should add them back in the actual model?